File: swift练习/Task 5/useUIButton/useUIButton/ViewController.m
//
//  ViewController.m
//  useUIButton
//
//  Created by 张皓 on 4/21/16.
//  Copyright © 2016 张皓. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    UIButton *btn = [[UIButton alloc] init];
    btn.frame = CGRectMake(0, 0, 400, 400);
    btn.backgroundColor = [UIColor redColor];
    [self.view addSubview:btn];
    
    [btn setImage:[UIImage imageNamed:@"1.jpg" ] forState:UIControlStateNormal];
    
    [btn setImage:[UIImage imageNamed:@"2.JPG" ] forState:UIControlStateHighlighted];
   
    [btn setTitle:@"normal" forState:UIControlStateNormal];
    
    [btn setTitle:@"Hight Lighted" forState:UIControlStateHighlighted];
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: swift练习/TASK 8/PlayVideo/PlayVideo/ViewController.swift
//
//  ViewController.swift
//  PlayVideo
//
//  Created by 张皓 on 16/5/21.
//  Copyright © 2016年 张皓. All rights reserved.
//

import UIKit
import MediaPlayer
import AVKit


class ViewController: UIViewController {

    
    
//    var moviePlayer: MPMoviePlayerViewController!
    
    
    var player: AVPlayerViewController!
    @IBAction func playVideoBtnClicked(sender: AnyObject) {
        
//        moviePlayer = AVMoviePlayerViewController(contentURL: NSBundle.mainBundle().URLForResource("video", withExtension: "mp4"))
        
//        moviePlayer = MPMoviePlayerViewController(contentURL: NSBundle.mainBundle().URLForResource("video", withExtension: "mp4"))
        
//        player = AVPlayerViewController(nibName: "video.mp4", bundle: NSBundle.mainBundle())
        
        player = AVPlayerViewController.init()
        player.player = AVPlayer(URL: (NSBundle.mainBundle().URLForResource("video", withExtension: "mp4"))!)
        
    
        
        
        
        if let movie = player{
            presentViewController(movie, animated: true, completion: nil)
        }
        
        
        
        
    }
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

